sdk_container/src/third_party/coreos-overlay/app-emulation/hv-daemons/hv-daemons-9999.ebuild

```
@@ -13,20 +13,18 @@ if [[ "${PV}" == 9999 ]]; then
 fi
 
 src_compile() {
-    # Build hv_vss_daemon, hv_kvp_daemon, hv_fcopy_daemon 
+    # Build hv_vss_daemon, hv_kvp_daemon, hv_fcopy_daemon
     kmake tools/hv
 }
 
 src_install() {
-    dobin "${S}/build/tools/hv/hv_fcopy_daemon"
-    dobin "${S}/build/tools/hv/hv_kvp_daemon"
-    dobin "${S}/build/tools/hv/hv_vss_daemon"
-
-    systemd_dounit "${FILESDIR}/hv_fcopy_daemon.service"
-    systemd_dounit "${FILESDIR}/hv_kvp_daemon.service"
-    systemd_dounit "${FILESDIR}/hv_vss_daemon.service"
-
-    systemd_enable_service "multi-user.target" "hv_fcopy_daemon.service"
-    systemd_enable_service "multi-user.target" "hv_kvp_daemon.service"
-    systemd_enable_service "multi-user.target" "hv_vss_daemon.service"
+    HV_DAEMONS=(hv_vss_daemon hv_kvp_daemon hv_fcopy_daemon hv_fcopy_uio_daemon)
+    for HV_DAEMON in "$HV_DAEMONS[@]"
```

Please make variables local, either by:

```suggestion
    local -a HV_DAEMONS=(hv_vss_daemon hv_kvp_daemon hv_fcopy_daemon hv_fcopy_uio_daemon)
    local HV_DAEMON
    for HV_DAEMON in "$HV_DAEMONS[@]"
```

or:

```suggestion
    local HV_DAEMON
    for HV_DAEMON in hv_vss_daemon hv_kvp_daemon hv_fcopy_daemon hv_fcopy_uio_daemon
```

---

Done.